pkg/catalog/repeater.go

```diff
@@ -40,7 +40,7 @@ func (mc *MeshCatalog) repeater() {
 }
 
 func (mc *MeshCatalog) handleAnnouncement(ann announcements.Announcement) {
-	if ann.Type == announcements.EndpointDeleted {
+	if ann.Type == announcements.PodDeleted {
```

Comment:
What about PodAdded and PodUpdated ?

Reply:
I decided - I only need to handle `PodDeleted` for now - to delete the cert that was issued for the given pod.
When the time comes for us to implement handling of the rest of the events - then we can add these...
But we'll continue to let them flow through the system they have been - pushing config to the Envoy proxies (sometimes unnecessarily)